packages/mobile/src/exchange/saga.test.ts

```diff
@@ -38,3 +43,37 @@ describe(doFetchTobinTax, () => {
       .run()
   })
 })
+
+describe(exchangeGoldAndStableTokens, () => {
+  it('makes the exchange', async () => {
+    const exchangeGoldAndStableTokensAction = {
+      makerToken: CURRENCY_ENUM.GOLD,
+      makerAmount: new BigNumber(SELL_AMOUNT),
+    }
+    await expectSaga(exchangeGoldAndStableTokens, exchangeGoldAndStableTokensAction)
+      .provide([
+        [call(getConnectedUnlockedAccount), true],
+        [
+          select(exchangeRatePairSelector),
+          {
+            goldMaker: '2',
+            dollarMaker: '0.5',
+          },
+        ],
+        [matchers.call.fn(sendTransaction), true],
+        [matchers.call.fn(sendAndMonitorTransaction), true],
+      ])
+      .put.like({
+        action: {
+          transaction: {
+            type: TokenTransactionType.Exchange,
+            inSymbol: CURRENCY_ENUM.GOLD,
+            inValue: SELL_AMOUNT.toString(),
+            outSymbol: CURRENCY_ENUM.DOLLAR,
+            outValue: (SELL_AMOUNT / 2).toString(),
+          },
+        },
+      })
+      .run()
+  })
+})
```

Comment:
Great that a test was added, however it doesn't catch the bug that this PR is fixing.
There are no assertions on the parameters passed to the `sendAndMonitorTransaction` function call.

Reply:
Yeah, sorry about that, just added it